.eslintrc.cjs

```diff
@@ -16,4 +16,15 @@ module.exports = {
     'quote-props': ['error', 'as-needed'],
     'comma-dangle': ['error', 'always-multiline'],
   },
+  overrides: [{
+    files: '*.spec.js',
+    plugins: [
+      'mocha',
+      'chai-friendly',
+    ],
+    extends: [
+      'plugin:mocha/recommended',
+      'plugin:chai-friendly/recommended',
+    ],
+  }],
```

Comment:
Linting for tests includes some extra rules to ensure we follow best testing practice! Don't worry too much about this configuration for now, you can just copy paste it around to other projects as you start them... that's usually how it goes :shrug: